packages/app/src/sections/futures/PositionType.tsx

```diff
@@ -6,23 +6,30 @@ import Body from 'components/Text/Body'
 type PositionProps = {
 	side: PositionSide
 	mobile?: boolean
+	variant?: 'trades' | 'default'
 }
 
-const PositionType: React.FC<PositionProps> = ({ side = PositionSide.LONG, mobile = false }) => {
+const PositionType: React.FC<PositionProps> = ({
+	side = PositionSide.LONG,
+	mobile = false,
+	variant = 'default',
+}) => {
 	return mobile ? (
 		<MobileStyledText side={side}>{side}</MobileStyledText>
 	) : (
-		<StyledText side={side}>{side}</StyledText>
+		<StyledText side={side} variant={variant}>
+			{side}
+		</StyledText>
 	)
 }
 
 const StyledText = styled(Body).attrs({ weight: 'bold', capitalized: true, type: 'span' })<{
 	side: PositionSide
+	variant: 'trades' | 'default'
```

Comment:
```suggestion
	variant: 'badge' | 'text'
```

This way the style is described for other components to use

Reply:
Yes. It's more understandable. Updated.